SAMtags.tex

```diff
@@ -182,15 +182,29 @@ \subsection{Additional Template and Mapping data}
 CIGAR string for mate/next segment.
 
 \item[MD:Z:\tagregex{[0-9]+(([A-Z]|\char92\char94[A-Z]+)[0-9]+)*}]
-String for mismatching positions.
+\hfill\\
+String encoding mismatched and deleted reference bases, used in conjunction with the {\sf CIGAR} and {\sf SEQ} fields to reconstruct the bases of the reference sequence interval to which the alignment has been mapped.
+This can enable variant calling without requiring access to the entire original reference.
+
+The {\tt MD} string consists of the following items, concatenated without additional delimiter characters:
+\begin{itemize}
+\item \verb"[0-9]+", indicating a run of reference bases that are identical to the corresponding {\sf SEQ} bases;
+\item \verb"[A-Z]", identifying a single reference base that differs from the {\sf SEQ} base aligned at that position;
+\item \verb"\^[A-Z]+", identifying a run of reference bases that have been deleted in the alignment.
+\end{itemize}
+
+As shown in the regular expression above, numbers alternate with the other items.
```

Comment:
I was initially writing a long comment on how the three separate regular expressions don't show this, and then noticed the un-edited part of the text with the full regexp.

Maybe edit this to "As shown in the full regular expression above".

Reply:
Good point. I'll make it “complete regular expression above” or something. Initially I had the bullet points as “Numbers …” / “Single bases …” / “`^`+string of bases …” rather than regexes so didn't notice this problem.